scalyr_agent/tests/configuration_test.py

```diff
@@ -1557,6 +1558,97 @@ def test_empty_substitution(self):
 
         self.assertEquals(config.api_key, "hi")
 
+    def test_print_config_doesnt_throw(self):
+        """Make sure that when we print the config options that we
+        don't throw any exceptions
+        """
+
+        self._write_file_with_separator_conversion("""{api_key: "hi there"}""")
+        mock_logger = Mock()
+        config = self._create_test_configuration_instance(logger=mock_logger)
+        config.parse()
+        try:
+            config.print_useful_settings()
+        except Exception as e:
```

Comment:
Don't bother to catch this here.  Just let the exception bubble up and fail the test.

Reply:
Done.